mapstore/geostore-datasource-ovr.properties

```diff
@@ -0,0 +1,24 @@
+#Default Postgres Connection values 
+geostoreDataSource.driverClassName=org.postgresql.Driver
+geostoreDataSource.url=jdbc:postgresql://database:5432/geostore

```

Comment:
By default, geOrchestra apps use the `georchestra` database & store their data in a schema named by the webapp name. Can we have the same here with a `mapstore` schema ?

Reply:
Definitely, as it is confusing and not helpful when setting schemas automatically where they match the app.. stumbled upon this one once again after setting up a mapstore instance